File: C++/level2/최댓값과_최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
//pass
using namespace std;

string solution(string s) {
    vector<int> numbers;
    string temp = "";

    for(auto c : s){
        if(c == ' '){
            numbers.push_back(atoi(temp.c_str()));
            temp = "";
        }
        else
            temp += c;
    }
    numbers.push_back(atoi(temp.c_str()));

    sort(numbers.begin(), numbers.end());

    return to_string(numbers[0]) + " " + to_string(numbers[numbers.size()-1]);
}

File: JavaScript/level2/최댓값과_최솟값.js
function sortNumber(a, b){
    return a-b;
}

function solution(s) {
    let arr = [];
    let val = "";
    
    for(let i of s){
        if(i === " "){
            arr.push(Number(val));
            val = "";
        }
        else
            val += i;
    }
    arr.push(Number(val));
    arr.sort(sortNumber);
    
    return String(arr[0]) + " " + String(arr[arr.length-1]);
}